lib/rubocop/path_util.rb

```diff
@@ -59,5 +60,18 @@ def self.pwd
     def self.reset_pwd
       @pwd = nil
     end
+
+    def hidden_file_and_not_hidden_dir?(pattern, path)
+      File.fnmatch?(
+        pattern, path,
+        File::FNM_PATHNAME | File::FNM_EXTGLOB | File::FNM_DOTMATCH
+      ) && File.basename(path).start_with?('.') && not_hidden_dir?(path)
+    end
+
+    def not_hidden_dir?(path)
```

Comment:
Negative predicates are always odd-looking. I think we should just name this `hidden_dir?` and invert the check in it.

Reply:
Sure, I totally agree! Thanks for the review.